extensions/vault/runtime/src/main/java/io/quarkus/vault/VaultKubernetesAuthService.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.vault;
+
+import java.util.List;
+
+import io.quarkus.vault.auth.VaultKubernetesAuthConfig;
+import io.quarkus.vault.auth.VaultKubernetesAuthRole;
+
+/**
+ * This services provides programmatic access to the Kubernetes auth method.
```

Comment:
```suggestion
 * This service provides programmatic access to the Kubernetes auth method.
```

Reply:
fixed and squashed. thanks for picking that up.

Reply:
Thank you! Gpod job! 😉